tests/test_basic_ops_dict.py

```diff
@@ -273,7 +273,7 @@ def test_map_expansion() -> None:
     def foo(a: int, b: int) -> int:
         return a + b
 
-    assert 12 == foo(**c)
+    assert 12 == foo(**c)  # type: ignore[misc]
```

Comment:
Please add a commend indicating this is known issue with mypy.

Reply:
Done.